### PR TITLE
enable pool test

### PR DESCRIPTION
**Pull request by catShaark**
Closes: #XXX
## What is the purpose of the change
Make test doesn't run the amm_test in pool-models, this PR fixes that

## Brief Changelog
* change name of TestCalculateAmountOutAndIn_InverseRelationship in internal/test_helpers
* add test function to run `BalancertestSuite` and `StableSwapTestSuite`

## Testing and Verifying
`Test_StableSwap_CalculateAmountOutAndIn_InverseRelationship` currently fails because of overflow issue in sdk math. So I commented out the test.

## Documentation and Release Note
* Does this pull request introduce a new feature or user-facing behavior changes? (yes / no)
* Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (yes / no)
* How is the feature or change documented? (not applicable   /   specification (`x/<module>/spec/`)  /  [Osmosis docs repo](https://github.com/osmosis-labs/docs)   /   not documented)

### CODE REVIEW

File: x/gamm/pool-models/balancer/pool_test.go [hunk 1]
```diff
@@ -30,6 +32,56 @@ var (
 	noErr           = false
 )
 
+type BalancerTestSuite struct {
```
Comment (mattverse) [inline x/gamm/pool-models/balancer/pool_test.go:35]: confirmed this is just a move of code, no extra logic

File: x/gamm/pool-models/stableswap/pool_test.go [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+type StableSwapTestSuite struct {
+	test_helpers.CfmmCommonTestSuite
+}
+
+func TestStableSwapTestSuite(t *testing.T) {
+	suite.Run(t, new(StableSwapTestSuite))
+}
+
+// func (suite *StableSwapTestSuite) Test_StableSwap_CalculateAmountOutAndIn_InverseRelationship() {
```
Comment (mattverse) [inline x/gamm/pool-models/stableswap/pool_test.go:19]: Why was this commented out?

Comment (catShaark) [reply x/gamm/pool-models/stableswap/pool_test.go:19]: because, stableswap math is not ready for testing, I think

Comment (mattverse) [reply x/gamm/pool-models/stableswap/pool_test.go:19]: sorry for the delay in review here, does this test not pass if we comment this out? (Asking this seeing that it wasn't commented out originally)

Comment (catShaark) [reply x/gamm/pool-models/stableswap/pool_test.go:19]: This test wasn't commented out before but it still did not run.

Comment (mattverse) [reply x/gamm/pool-models/stableswap/pool_test.go:19]: Hrrm what's the reason it didn't run? Sorry if Im missing something here

**Pull request closed without merging**